sanity/doctors/Doctor.js

```diff
@@ -1,5 +1,15 @@
+import { Log } from 'decentraland-commons'
+
 export class Doctor {
+  constructor() {
+    this.log = new Log(this.constructor.name)
+  }
+
   async diagnose(options) {
     throw new Error('Not implemented')
   }
+
+  logErrors(faultyAssets) {
+    this.log.error(faultyAssets.map(({ error }) => error).join('\n'))
```

Comment:
I know this way of looping and logging is way faster than 

`faultyAssets.forEach({errorr} => this.log.error(error))`

But if we have a lot of errors, more memory will be needed. 

What do you think of adding a comment saying that memory will be needed _in a linear way_ related to the count of errors

Reply:
Huh, haven't thought of that you're right. I'll add the comment but if you think this might be a problem we can change it to 

```javascript
faultyAssets.forEach(({error}) => this.log.error(error))
```